src/transformers/modeling_utils.py

```diff
@@ -4027,7 +4029,7 @@ def from_pretrained(
         gguf_file = kwargs.pop("gguf_file", None)
         tp_plan = kwargs.pop("tp_plan", None)
         tp_size = kwargs.pop("tp_size", None)
-        key_mapping = kwargs.pop("key_mapping", None)
+        key_mapping = kwargs.pop("key_mapping", cls._key_mapping)
```

Comment:
And so only use it here if it exists
```suggestion
        key_mapping = kwargs.pop("key_mapping", getattr(cls, "._key_mapping", None))
```

But just a bit worried if it could become an issue when saving then reloading?